### PR TITLE
Move to edge-prefix for auth/router

### PR DESCRIPTION
**Pull request by alexellis**
Signed-off-by: Alex Ellis <alexellis2@gmail.com>

## Description

This change syncs with the latest changes for the router and auth
services which now have a prefix of edge-.

## How Has This Been Tested?

TBD once upstream changes are merged in openfaas-cloud repo

## Checklist:

I have:

- [x] checked my changes follow the style of the existing code / OpenFaaS repos
- [ ] updated the documentation and/or roadmap in README.md
- [x] read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] signed-off my commits with `git commit -s`
- [ ] added unit tests

### CODE REVIEW

**Comment by matipan** [comment]
I used the binary of this branch to deploy to a GKE cluster and it worked ok. I recently did a push and build went ok: https://github.com/matipan/faas-demo/runs/123482699
Authentication is working ok too

File: templates/edge-auth-dep.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ spec:
 # Update for your configuration:
           - name: client_secret # this can also be provided via a secret named of-client-secret
             value: ""
-          - name: client_id
+         - name: client_id
```
Comment (matipan) [inline templates/edge-auth-dep.yml]: This needs to be fixed. The space was ok

**Comment by alexellis** [comment]
Thank you. I will cut a binary now by doing a release

File: templates/edge-auth-dep.yml (hunk 1 above)
Comment (alexellis) [reply templates/edge-auth-dep.yml]: Please can you take that?

**Pull request merged**